Tests/SpotifyWebAPIMainTests/API Tests/SpotifyAPIPlaylistsTests.swift

```diff
@@ -1554,7 +1554,15 @@ extension SpotifyAPIPlaylistsTests where
         XCTAssertFalse(
             Self.spotify.authorizationManager.isAuthorized(for: [])
         )
-        Self.authorizeAndWaitForTokens(scopes: Scope.playlistScopes)
+        let randomScope = Scope.allCases.randomElement()!
```

Comment:
randomElement sounds like a good unit test until you encounter a semi-reproducible unit somewhere and can't track it down. I would always advise to either iterate over each, or pick one or two specific ones to test

Reply:
You're right that if this test failed I wouldn't know which scope caused the failure. I can change this by passing the scope to the `message` parameter of `XCTAsertFalse`.